test/sagemaker_tests/tensorflow/tensorflow2_training/integration/sagemaker/test_mnist.py

```diff
@@ -77,11 +77,12 @@ def test_distributed_mnist_no_ps(sagemaker_session, ecr_image, instance_type, fr
 @pytest.mark.integration("parameter server")
 def test_distributed_mnist_ps(sagemaker_session, ecr_image, instance_type, framework_version):
     resource_path = os.path.join(os.path.dirname(__file__), '..', '..', 'resources')
-    script = os.path.join(resource_path, 'mnist', 'mnist_estimator.py')
+    script = os.path.join(resource_path, 'mnist', 'mnist_estimator2.py')
```

Comment:
does this support TF<2.6 like TF1.15, TF2.4?

Reply:
It won't support tf1.15 for sure. It will support tf>=2.0